indico/modules/events/editing/controllers/base.py

```diff
@@ -92,8 +92,10 @@ class RHContributionEditableBase(RequireUserMixin, RHContributionDisplayBase):
     _editable_query_options = ()
 
     def _check_access(self):
-        RequireUserMixin._check_access(self)
-        RHContributionDisplayBase._check_access(self)
+        self.is_service_call = TokenAccessMixin._token_can_access(self)
```

Comment:
What do you think about moving this to `TokenAccessMixin`?

`is_service_call = False` on the class level (so it's always set even if `TokenAccessMixin._token_can_access` is not executed, and set it to true in `_token_can_access`

Reply:
And then just calling `TokenAccessMixin._token_can_access(self)` and expect the `is_service_call` to be assigned?

Reply:
In here: `if not TokenAccessMixin._token_can_access(self):`

and just use the attribute when you need to check it elsewhere

Reply:
I like the idea of simplifying this service call assignment. But for some reason, it feels dangerous to "hide" this assignment (or implicitly set it). Nevertheless, since it will get re-executed every time on `_check_access` I think it is fine.